web/pdf_find_controller.js

```diff
@@ -276,7 +253,7 @@ var PDFFindController = (function PDFFindControllerClosure() {
         page.textLayer.updateMatches();
       }
       //scroll match into view
-      if (PDFJS.multiple !== undefined) {
+      if (PDFJS.multiple == undefined) {
```

Comment:
Compare with === 
To compare undefine type use typeof:
if (typeof PDFJS.multiple === 'undefined')